src/components/alert/alert.stories.js

```diff
@@ -0,0 +1,139 @@
+import { html } from "lit-html";
+import "../button";
+import "../alert";
+import styles from "./styles.scss";
+import { stylesheet } from "../../../.storybook/utils.js";
+
+export default {
+  title: "custom-elements/vl-alert",
+  decorators: [(story) => html`${stylesheet(styles)}${story()}`],
+  args: {
+    title: "Opgelet!",
+    icon: "warning",
+    size: "normal",
+    type: "info",
+    closable: false,
+  },
+  argTypes: {
+    title: {
+      name: "data-vl-title",
+      type: { summary: "string" },
+      description: "Attribuut wordt gebruikt om de titel te bepalen.",
+      table: {
+        defaultValue: { summary: "" },
+      },
+    },
+    icon: {
+      name: "data-vl-icon",
+      type: "select",
+      options: ["warning", "check", "info-circle"],
+      description: "Attribuut wordt gebruikt om het icoon type te bepalen.",
+      table: {
+        type: { summary: "string" },
+        defaultValue: { summary: "" },
+      },
+    },
+    size: {
+      name: "data-vl-size",
+      type: "select",
+      options: ["normal", "small"],
+      description:
+        "Attribuut activeert een variant van de waarschuwing maar kleiner.",
+      table: {
+        type: { summary: "string" },
+        defaultValue: { summary: "" },
+      },
+    },
+    type: {
+      name: "data-vl-type",
+      type: "select",
+      options: ["info", "success", "warning", "error"],
+      description:
+        "Attribuut bepaalt de soort van waarschuwing, foutmelding, probleemmelding of succesmelding.",
+      table: {
+        type: { summary: "string" },
```

Comment:
Als je maar een aantal strings kan meegeven, vermeld je die bij de summary.

voorbeeld: https://uig-webcomponents.omgeving.vlaanderen.be/?path=/docs/custom-elements-vl-accessibility--default data-vl-compliance of data-vl-evaluation

Reply:
Fixed